### ISSUE TITLE
Replicate VBA IntelliSense behavior in twinBASIC IDE

### ISSUE THREAD

**Issue by mwolfe02** [WaynePhillipsEA/twinbasic#779]
## Intellisense Taming

v0.10.4966 adds many intellisense tweaks.  Most notably, intellisense now won't preselect a partial match.  Instead, you'll see the partial text you've entered as the top item in the list:
<img width="626" alt="tb_IntellisenseImplements3" src="https://user-images.githubusercontent.com/25738109/130632827-46785caf-f47e-4fb0-b768-6be29e03afb3.png">

This means that you shouldn't accidently trigger wrong intellisense matches during general editing, and this also helps because we can now turn on SPACE bar support for confirming the intellisense selection, like in VBx (also done in v0.10.4966).

_Originally posted by @WaynePhillipsEA in https://github.com/WaynePhillipsEA/twinbasic/issues/337#issuecomment-904684941_

**Comment by WaynePhillipsEA** [comment]
I agree, and I suspect we can work to get this much closer to the VBx offering now.
